config/config.go

```diff
@@ -67,3 +71,41 @@ func LoadConfig(configFile string) {
 		log.Info().Msg("Config file validated!")
```

Comment:
:wave: please do not forget to add log.Trace() to expose the full de-serialized config